README.md

```diff
@@ -18,6 +18,10 @@ If you have a suggestion or question, open an [issue](https://github.com/redhat-
 
 See the [Contributing guide](CONTRIBUTING.md) for information on how to contribute an update.
```

Comment:
Such a tiny thing but it should be 'information about' not 'information on' to be fully IBM-style compliant. Same for the Administrator responsibilities below.
Overall looks great to me though! Thanks @inoxx03

Reply:
@ndeevy Very true:) Thank you. Although, I don't think that I added or changed this line in my current PR, let me fix it anyway:)

Reply:
```suggestion
See the [Contributing guide](CONTRIBUTING.md) for information about how to contribute an update.
```